### DISCUSSION TITLE
I need help logging into my account

### DISCUSSION THREAD

**Discussion by heiyehk** [npm/feedback#723] [Website] [answered]
I need help logging into my account
I can't log in. I don't remember Enter One-time Password, and I can't log in even if I try to retrieve the password and change it. At present, the login information is not retained after the computer is reinstalled. Tried the support page and reported an error.

**Comment by lovanya** [comment]
I have the same problem, I lost my email, so I can't get One-time Password! What should I do now???

**Reply by MylesBorins** [reply]
Please open a ticket with support. If you no longer have access to the email and we cannot verify your identity otherwise we may not be able to restore access to your account.

**Reply by lovanya** [reply]
I tried to contact u guys on https://www.npmjs.com/support?inquire=account, but I failed. Don't know what's wrong?
After I clicked the submit button, I got this:
![image](https://user-images.githubusercontent.com/16385444/178307943-b4c6e31f-741f-4e85-b57c-98864a578a07.png)

**Reply by MylesBorins** [reply]
I'll surface this to our engineering team. Can you try another browser or potentially in incognito mode

**Reply by heiyehk** [reply]
I've visited and tried the support page several times, and I keep getting errors, even in incognito mode.

**Reply by lovanya** [reply]
I finally know how to solve it. :)

Seems there is a program to check if ur browser environment is secure. So we can solve it in the following ways:
1. Back to the home page and click the **Sign up for free** button.
2. Pass the man-machine verification procedure, then u will see the sign up page.
3. Back to support page to submit ur issues.

**Comment by MylesBorins** [comment, marked as answer]
Please try the support page again, that is the only way to attempt to regain access to your account through identity verification. I'll follow up with the engineering team on these errors.

**Reply by heiyehk** [reply]
It's been a long time, but the page still shows the error loading page.

**Reply by MylesBorins** [reply]
have you tried in a different browser or with incognito mode? Can you share the following details:

* what browser + version
* a screenshot of any errors

**Reply by narula0781** [reply]
@ghostdevhv - Can you look into this ? Thanks

**Reply by heiyehk** [reply]
@MylesBorins 
![image](https://user-images.githubusercontent.com/33891067/181694700-ccbf6657-1ba1-4977-b4b1-e389b8aefbfd.png)
I have used incognito mode but it still shows this error.

**Reply by heiyehk** [reply]
windows version: 21H2
browser version: 103.0.1264.71

**Reply by ghostdevhv** [reply]
@heiyehk - is there any error in the console?

**Reply by heiyehk** [reply]
> @heiyehk - is there any error in the console?

yes.
```
POST https://www.npmjs.com/support 403

load-path.js:46 
        
e.exports @ load-path.js:46
value @ router.js:95
value @ router.js:73
value @ form.js:42
value @ contact.js:105
value @ generic.js:25
onSubmit @ generic.js:79
value @ form.js:10
onSubmit @ form.js:22
s @ react-dom.production.min.js:14
h @ react-dom.production.min.js:14
(匿名) @ react-dom.production.min.js:14
v @ react-dom.production.min.js:15
it @ react-dom.production.min.js:52
at @ react-dom.production.min.js:51
st @ react-dom.production.min.js:52
ht @ react-dom.production.min.js:56
I @ react-dom.production.min.js:287
B @ react-dom.production.min.js:19
Xt @ react-dom.production.min.js:70
Jt @ react-dom.production.min.js:69
t.unstable_runWithPriority @ scheduler.production.min.js:19
Fa @ react-dom.production.min.js:122
P @ react-dom.production.min.js:287
$t @ react-dom.production.min.js:68
store.js:37 
        
       SyntaxError: Unexpected token < in JSON at position 0
    at JSON.parse (<anonymous>)
    at load-path.js:83:27
    at d.onreadystatechange (load-path.js:51:22)
    ```

**Reply by lovanya** [reply]
I got the same errors. 
![image](https://user-images.githubusercontent.com/16385444/182273565-9a07b89c-71be-44ca-9600-8d3c38605416.png)
![image](https://user-images.githubusercontent.com/16385444/182273655-9f8c71a4-0092-4c1a-ac82-c70f55a8e4b2.png)

**Reply by heiyehk** [reply]
> I got the same errors. ![image](https://user-images.githubusercontent.com/16385444/182273565-9a07b89c-71be-44ca-9600-8d3c38605416.png) ![image](https://user-images.githubusercontent.com/16385444/182273655-9f8c71a4-0092-4c1a-ac82-c70f55a8e4b2.png)

I tried to send the email today and it has been sent successfully!
